README.md

```diff
@@ -146,11 +146,11 @@ We've also included an experimental script to load in the `.spacy` binary format
 
 ```
 pip install -r requirements_train.txt
-python -m prodigy data-to-spacy ./data/annotations2 --ner my_ner_data -es 0
-python scripts/train_hf_ner.py ./data/annotations2/train.spacy ner-model
+python -m prodigy data-to-spacy ./data/annotations2 --ner my_ner_data
+python scripts/train_hf_ner.py ./data/annotations2/train.spacy ./data/annotations2/dev.spacy ner-model
```

Comment:
The biggest advantage of splitting out beforehand is that we can show the three steps: `data-to-spacy`, then either `spacy train` or `train_hf_ner.py`, or both. So I wouldn't repeat the `data-to-spacy` command here, and it allows us to get rid of this ugly `annotations2` dir.

At L138, I'd rewrite slightly so that the `bash` code bit gets split into two, one for `data-to-spacy` and one for `spacy-train`, making it clear to readers how they can combine them.

Reply:
Also I'd like to keep the `-es` parameter in the `data-to-spacy` command as users often will want to control the % of the split.